angular-app/angular.json

```diff
@@ -23,7 +23,8 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.css"
+              "src/styles.css",
+              "src/status-theme.scss"
```

Comment:
We should gradually switch to the name **Explorer**, so please change that here.

Reply:
Should I rename status to explorer everywhere? or do it in the next PR?

Reply:
Fine in another PR as well.